conf/es/spain01.yml

```diff
@@ -404,7 +404,7 @@ years:
       en: Día del Trabajador
       es: Día del Trabajador
   - public_holiday: true
-    date: '2021-05-02'
+    date: '2021-05-3'
```

Comment:
```suggestion
    date: '2021-05-03'
```

Reply:
Oh my bad, thanks Tim - not sure how to accept the suggested change here?

Reply:
I'll do it. For next time it's the `Commit suggestion` dropdown:

<img width="713" alt="Screenshot 2021-03-11 at 16 25 16" src="https://user-images.githubusercontent.com/3686887/110819760-6401cc00-8286-11eb-9ee4-b906507a7466.png">